### DISCUSSION TITLE
Optimal Configuration for Best OCR Output Using Scribe.js

### DISCUSSION THREAD

**Discussion by zakariamehbi** [scribeocr/scribe.js#11] [Q&A]
Hello,

I noticed that the current documentation lacks details in this area, so I’d like to directly ask the team for those seeking the best OCR results.

What would be the ideal configuration for Scribe.js to achieve the highest quality OCR output on a PDF, without considering performance or execution time constraints?

Thank you in advance for your help!

**Comment by Balearica** [comment]
The default settings already optimize for quality.

1. The default recognition mode is already `quality` rather than `speed`. 
2. For `.pdf` files, the `extractText` function extracts existing text content rather than running OCR for text-native files, however ignores existing text content (and runs OCR instead) for image-native files with an existing OCR layer.
   1. This results in the highest quality result for files where Scribe.js is more accurate than whatever program created the original layer, which is generally true.
